changelog/2022-03-21T11_09_46-05_00_concurrent_normalization

```diff
@@ -0,0 +1 @@
+CHANGED: Add concurrent normalization flag [#2074](https://github.com/clash-lang/clash-compiler/pull/2074)
```

Comment:
Drive by review: Please explicitly name the flag (`-fclash-concurrent-normalization`) in the Changelog, otherwise people still have to hunt for it.

Reply:
As well as a brief description of what it does. The Changelog should be an understandable stand-alone unit. It doesn't have to be extensive, but it should be informative.

Reply:
Also, it should be `ADDED`, not `CHANGED`. It would probably be good to document somehwere the keywords we expect here.

Reply:
I started documenting changelog best practices in #2169.